aspnetcore/mvc/views/tag-helpers/builtin.md

```diff
@@ -11,7 +11,7 @@ Microsoft has included 17 Tag Helpers that can be immediately used in .net Core
 
 **[AnchorTagHelper](builtin/resources/AnchorTagHelper.md)**
 
-[comment]: **[CacheTagHelper](builtin/resources/AnchorTagHelper.md)**
+**[CacheTagHelper](builtin/resources/CacheTagHelper.md)**
```

Comment:
Hey @Rick-Anderson , could you give a quick opinion on this.  @GuardRex makes a good point that in the forms doc all the tag helpers are expanded out (Cache Tag Helper, not CacheTagHelper).  I always do it the CacheTagHelper way but happy to change all my references, just want to make sure I get it right.  -thanks